apps/wallet-mock/src/support/event-trigger.ts

```diff
@@ -0,0 +1,58 @@
+// For now it supports only simple transaction review event, ideally it should be extended to support all possible workflows
+
+Cypress.Commands.add('triggerTransactionEvent', (event, data) => {
+  cy.window().then((win) => {
+    win.document.body.dispatchEvent(
+      new CustomEvent('new_interaction', {
+        detail: {
+          name: 'INTERACTION_SESSION_BEGAN',
+          data: { workflow: 'TRANSACTION_REVIEW' },
+          traceID: '1',
```

Comment:
Trace id's should be unique to mirror backend, can we use nanoid to generate these